hindi/art-of-living/who_developed_hindi.md

```diff
@@ -0,0 +1 @@
+Hindi (Hindi: हिन्दी hindī), sometimes spelled हिंदी, also called Modern Standard Hindi (Hindi: मानक हिन्दी mānak hindī), is a standardised and Sanskritised register of the Hindustani language. Hindi is one of the many official [[official languages |language]] of the Union of India.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.